wiki/entry_kreischer.qmd

```diff
@@ -0,0 +1,41 @@
+
+The study uses many of these findings as a basis. It further adds to a mixed field of opinions within the research. While some researchers have found that interpersonal digital communication (on Facebook) has indeed fulfilled the feeling of autonomy to a high degree, other researchers have come to different conclusions. Thus stating, that a felt overload of communication, a use of many digital communication platforms at once or an unwanted interruption from something else can induce stress and the feeling of autonomy-loss.
+
+Considering all of the aspects mentioned above, this specific study adds to the knowledge and connection between digital media use, specifically in communication, and the importance of experienced autonomy. New insights are being formed by focusing on individual and private communication via digital media, whereas prior studies had primarily focused on the aspect of felt autonomy and digital media in a work related environment. Additionally, this study differentiates between autonomy frustration and satisfaction, something that has also not been done in many studies before. Going from there, with the findings of this specific study, many others can build upon that knowledge and go further in depth. For example examining the key factors leading to experience of autonomy loss or other questions raised by the study, as can be seen in the following paragraph.
+
+Questions raised by the study
+
+Whilst the study answers some questions concerning the positive and negative impact of satisfying the feeling of autonomy within digital media, it does not answer, what exactly triggers the feeling of autonomy loss. This however was being done by other studies following. As an example, the study done by Gilbert et al. (2022) can be mentioned. The scientists first of all supported the main finding of this study, confirming the loss of autonomy while using digital media to enhance stress in real life, worsening the overall mental health of the user. This was especially the case, when there was a big communication load on the user. This however was not connected to being connected on several platforms at once. Gilberts study being a good example building upon the knowledge formed in this study, as was already mentioned above. Going further in depth with this first question raised, a follow up study done to examine further media development on that field would also be interesting. How does the new development of AI for example influence experienced autonomy when using digital media as it becomes smarter and more self-functioning?
+
+Another question raised is why people still use digital media, even if it enhances their stress perception and deteriorates their overall well-being. This being rather in the field of general media usage research.
+
```

Comment:
Can you identify facets of digital autonomy that are not covered in this paper?

Reply:
Whilst Meier solely focuses on digital autonomy concerning one individual and ones own actions, other have also considered digital autonomy of an individual in a group setting, for example in fanfiction groups. The aspect important for autonomy in this case is how self-empowered one can be in a group setting. Further, informational autonomy is added. This meaning, that the feeling of autonomy is strengthened if an individual has the idea of being well informed through digital media and having the choice of where to inform themselves. Another point that is mentioned in other research was how one has the ability to keep their privacy online.

Reply:
Regarding your question about why people use digital media in general: 
The effects of social media are individual for everyone and not everyone experiences negative aspects. This is indeed an interesting topic for future research and should be explored further. Questions like 'Are there differences in unfulfilment when it comes to character traits' could be asked. 


Regarding your question about whether AI will affect autonomy: 
I can understand the meaning behind the question, but I don't think AI is responsible for fulfilling your individual autonomy needs. It may create new aspects or ideas of why you might feel loved or sad, but it will not satisfy your needs. 

All in all, your article gives a good overview of the topic.